pom.xml

```diff
@@ -57,6 +57,10 @@
             1.4-SNAPSHOT
         </version.osiam.addon-self-administration-plugin-example>
         <version.osiam.connector4java>1.7-SNAPSHOT</version.osiam.connector4java>
+
+        <debug.timeout.connect>10000</debug.timeout.connect>
+        <debug.timeout.read>30000</debug.timeout.read>
```

Comment:
If you set the default values with System.getProperties in the Abstract* classes, you don't need to set them here and can remove both lines